lib/go/test/tests/protocol_mock.go

```diff
@@ -1,513 +1,703 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements. See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership. The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License. You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-// Automatically generated by MockGen. DO NOT EDIT!
-// Source: thrift (interfaces: TProtocol)
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Since we updated `TProtocol` and have to rerun mockgen, I keep this file to be the exact output of mockgen, and thus removed the license header.

@Jens-G I'm not 100% sure if that's in compliance to Apache license policy (this file is purely machine generated). if that's not allowed, I can add the license header back.